content/docs/welcome-guide/setup/installing-linux.md

```diff
@@ -109,14 +109,22 @@ Example of launching Project Manager from the shell:
 ## Installing O3DE from a Snap package
 
 {{< note >}}
-The Snap package is experimental and may run into issues on some distros. These instructions have been tested on Ubuntu 20.04 and 22.04 LTS.
+The Snap package is experimental and may run into issues on some distros. These instructions have been tested on Ubuntu 20.04 and 22.04 LTS. You will need to install all of the dependencies listed in [Linux requirements](https://www.o3de.org/docs/welcome-guide/requirements/#linux).
```

Comment:
LF is switching the URL for docs to docs.o3de.org, so for this link to work properly, it will need to be a relative path.
```suggestion
The Snap package is experimental and may run into issues on some distros. These instructions have been tested on Ubuntu 20.04 and 22.04 LTS. You will need to install all of the dependencies listed in [Linux requirements](/docs/welcome-guide/requirements/#linux).
```